docs/interceptors.md

```diff
@@ -565,4 +609,4 @@ field both in the body of the payload as well as via the extra fields added to t
 
 ## Implementing custom `Interceptors`
 
-Tekton Triggers ships with the `ClusterInterceptor`and `Interceptor` Custom Resource Definition (CRD), which you can use to implement custom `Interceptors`. See [`ClusterInterceptors`](./clusterinterceptors.md) and [`NamespacedInterceptors`](./namespacedinterceptors.md)  for more information.
+Tekton Triggers ships with the `ClusterInterceptor`and `Interceptor` Custom Resource Definition (CRD), which you can use to implement custom `Interceptors`. See [`ClusterInterceptors`](./clusterinterceptors.md) and [`NamespacedInterceptors`](./namespacedinterceptors.md)  for more information.
```

Comment:
Whats the change here :thinking:

Reply:
I don't see any change here either and was surprise with this difference here. After looking deeply the only difference I could figure out is there is an extra line in the current triggers main interceptors.md file. Trying to put that extra line back to see if that helps remove this unwanted change showing